docs/userguide/envvars.md

```diff
@@ -0,0 +1,103 @@
+# Environment Variables
+
+Several aspects of BayBE can be configured via environment variables.
+
+## Basic Instructions
+Setting an environment variable with the name `ENVVAR_NAME` is best done before calling
+any Python code, and must also be done in the same session unless made persistent, e.g.
+via `.bashrc` or similar.
+```bash
```

Comment:
Not an expert here, but does this example work for all OS? I only remember that setting environment variables on Windows is always a pain and I am not sure if it works like that. So maybe skip this explicit example and just mention something like "note that it depends on your OS how to set environment variables"?

Reply:
yes Windows console would be different (its not a bash based console),  its a fairly triial amtter ot dinf out how to chagne env vars on windows but I added an instruction